brave/browser-laptop
# Don't prompt for restart when changing Toolbar and UI element scale (…

…since the underlying problem was fixed)

Fixes https://github.com/brave/browser-laptop/issues/10089

Auditors: @luixxiul, @alexwykoff

Submitter Checklist:


Test Plan:
1. Go to Preferences > Advanced
2. Under "Toolbar and UI Elements scale", pick a different value from the drop down
3. you'll be prompted to restart. Ignore this or click "no"
4. Notice that the top area (where the URL bar is) has changed in size
5. Notice that you are no longer prompted to restart

Reviewer Checklist:

Tests
